pkg/collector/corechecks/snmp/metadata/payload.go

```diff
@@ -0,0 +1,39 @@
+package metadata
+
+// PayloadMetadataBatchSize is the number of resources per event payload
+// Resources are devices, interfaces, etc
+var PayloadMetadataBatchSize = 100
+
+// NetworkDevicesMetadata contains network devices metadata
+type NetworkDevicesMetadata struct {
+	Subnet           string              `json:"subnet"`
+	Devices          []DeviceMetadata    `json:"devices,omitempty"`
```

Comment:
When is there ever multiple devices here?

Reply:
On only send one device for now, but it might be possible in the future. It's just to be future proof without much effort.